packages/@webex/plugin-meetings/src/multistream/remoteMediaManager.ts

```diff
@@ -490,6 +490,16 @@ export class RemoteMediaManager extends EventsScope {
     return this.currentLayoutId;
   }
 
+  /**
+   * sets the preferLiveVideo
+   */
+  public setPreferLiveVideo(preferLiveVideo: boolean) {
+    this.config.video.preferLiveVideo = preferLiveVideo;
```

Comment:
I think it would be useful to add a log here so that we know when this API is called - this may be relevant when debugging issues